### ISSUE TITLE
Reference/register the `webhook-` HTTP header fields

### ISSUE THREAD

**Issue by dret** [standard-webhooks/standard-webhooks#107]
I am wondering: Are the `webhook-` HTTP header fields defined somewhere? If so it would be great to reference that definition. If not it might make sense to consider defining these and then adding them to the IANA registry of HTTP header fields.

**Comment by tasn** [comment]
Not at the moment, but it sounds like a great idea.

**Comment by dret** [comment]
On https://webconcepts.info/concepts/http-header/ I have a collection of standardized and other well-known header fields. I guess I could add the three fields from the spec and add the spec as the definition, right?

**Comment by dret** [comment]
Also, if you need input/help with defining the fields in a slightly more IANA-friendly way, please let me know. I like standards.

**Comment by tasn** [comment]
Yeah, both things would be great. Both adding it to the list, and defining them in a more IANA-friendly way.

**Comment by dret** [comment]
On 2024-01-26 19:47, Tom Hacohen wrote:
> Yeah, both things would be great. Both adding it to the list, and defining them in a more IANA-friendly way.

here's a first version: https://webconcepts.info/specs/Webhook/standard/webhooks

it would be really useful if each of the header fields would have a complete description of what it's doing. the current descriptions are rather minimal and i don't just want to make things up.
